resources/Materials/TestSuite/stdlib/texture/tokenGraph.mtlx

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+   <nodegraph name="Tokenized_Image" >
+      <token name="Image_Resolution" type="string" uniform="true" value="2k" uiname="Image Resolution" />
+      <token name="Image_Extension" type="string" uniform="true" value="png" uiname="Image Extension" />
```

Comment:
Will remove. It was actually done on save of this file.

Reply:
Ah yes since the tokens are represented as inputs it writes all input attributes to the file. Not a big issue but we could fix it by adding an attribute ignore list for tokens.